llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -1016,6 +1016,39 @@ def HasStdExtSmctrOrSsctr : Predicate<"Subtarget->hasStdExtSmctrOrSsctr()">,
                                "'Smctr' (Control Transfer Records Machine Level) or "
                                "'Ssctr' (Control Transfer Records Supervisor Level)">;
 
+// Packed SIMD Extensions
+def FeatureStdExtP
+    : RISCVExperimentalExtension<1, 0,
```

Comment:
I guess that should be 0.12, although the spec define the version in a uncommon way...(012 rather than 0.12)